### PR TITLE
updates instructions based on RSLondon website and newer git version

### LINKED ISSUE #1: Check that git for windows instructions are up-to-date
Based on David PS's comment on the word file:
```
these steps were for 2.28 and now they are at 2.31. I've looked through the release notes and I've not seen anything obvious that may have change these steps. Maybe the GCM Core is not "new" anymore or that option is not there as it's the default now.
The SwCarpentry instructions includes a step about the branch naming. - https://rslondon.github.io/2021-05-20-rslondon-python/#shell
```

### PR DESCRIPTION
**Pull request by alessandrofelder**
Closes #1

### CODE REVIEW

File: _pages/setup/instructions_win.md [hunk 1]
```diff
@@ -10,19 +10,24 @@ instructions below:
 
 1. Download the [Git for Windows installer](https://git-for-windows.github.io/).
 1. Run the installer and follow the steps below:
+Ensure that "Git Credential Manager Core" is selected and click on "Next".
+Ensure that "Enable file system caching" is selected and click on "Next".
+Click on "Install".
+Click on "Finish" or "Next".
```
Comment (ageorgou) [inline _pages/setup/instructions_win.md]: These are present later on?

File: _pages/setup/instructions_win.md [hunk 2]
```diff
@@ -10,19 +10,24 @@ instructions below:
...
    - Ensure that "Use the OpenSSL library" is selected and click on "Next".
    - Keep "Checkout Windows-style, commit Unix-style line endings" selected and click on "Next".
    - **Ensure that "Use Windows' default console window" is selected and click on "Next".**
    - Ensure that "Default (fast-forward or merge) is selected and click "Next". 
-   - Ensure that the (New!) "Git Credential Manager Core" is selected and click on "Next". 
+   - Ensure that "Git Credential Manager Core" is selected and click on "Next".
    - Ensure that "Enable file system caching" is selected and click on "Next". 
-   - Experimental features: Keep the default (i.e., nothing selected) - click "Next"
    - Click on "Install".
    - Unselect "View release notes" (unless you are interested), and click on "Next".
```
Comment (ageorgou) [inline _pages/setup/instructions_win.md]: Actually, should this be `click on "Finish"` (or `"Finish or Next"`)?

**Review by ageorgou — CHANGES_REQUESTED** [review]

File: _pages/setup/instructions_win.md (hunk 2 above)
Comment (alessandrofelder) [reply _pages/setup/instructions_win.md]: I've tested, and it should be "Finish" (at least according to what happens locally on native Win for me).

File: _pages/setup/instructions_win.md (hunk 1 above)
Comment (alessandrofelder) [reply _pages/setup/instructions_win.md]: Woops 🤦

**Comment by alessandrofelder** [comment]
Thanks for catching my silly mistakes @ageorgou ! I've manually tested the instructions locally and fixed now.

**Review by ageorgou — APPROVED** [review]

**Pull request merged**